example/deisa_client.py

```diff
@@ -0,0 +1,26 @@
+from deisa import Deisa
+
+# Scheduler file name and configuration file
+scheduler_info = '/tmp/scheduler.json'
+
+# Initialize Deisa
+deisa = Deisa(scheduler_info, nb_workers=1, use_ucx=False)
+
+# either: Get data descriptor as a list of Deisa arrays object
+arrays = deisa.get_deisa_arrays()
+
+# Select data
+gt = arrays["global_t"][...]
+
+# Check contract
+arrays.check_contract()
+
+
+##########################
```

Comment:
Shall we add a small analysis example here?

Reply:
I believe it's a good idea, and could deserve its dedicated issue & PR